Server/ConcordiaCurriculumManagerTest/UnitTests/Services/CourseServiceTest.cs

```diff
@@ -49,7 +49,32 @@ public async Task InitiateCourseCreation_CourseExists_ThrowsArgumentException()
     {
         courseRepository.Setup(cr => cr.GetCourseBySubjectAndCatalog(It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync(GetSampleCourse());
 
-        await courseService.InitiateCourseCreation(GetSampleCourseCreationInitiationDTO(), GetSampleUser());
+        await courseService.InitiateCourseCreation(GetSampleCourseCreationInitiationDTO(), GetSampleUser(), Guid.NewGuid());
+    }
+
+    [TestMethod]
+    [ExpectedException(typeof(Exception))]
+    public async Task InitiateCourseCreation_DossierDoesNotExist_LogsAndThrowsException()
+    {
+        courseRepository.Setup(cr => cr.GetCourseBySubjectAndCatalog(It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync((Course?)null);
```

Comment:
Did you want to mock the dossierRepository here, going by the method name?

Reply:
Yes you're right, I was just confused because I saw the mock for GetCourseBySubjectAndCatalog